python/tvm/script/builder/tir/base.py

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""TVM Script TIR Frame"""
+from tvm._ffi import register_object as _register_object
+
+from ..frame import Frame
+
+@_register_object("script.builder.tir.TIRFrame")
+class TIRFrame(Frame):
+    def __enter__(self) -> "TIRFrame":
+        return self
+
+    def __exit__(self, ptype, value, trace) -> None:
+        pass
```

Comment:
no need for those two methods which are both defined in `Frame`, just do:

```suggestion
@_register_object("script.builder.tir.TIRFrame")
class TIRFrame(Frame):
    ...
```